CODEOWNERS

```
@@ -1 +1 @@
-*   @databricks/field-dev-ecosystem @databricks/eng-plat-auto-exp-reviewers
+*   @databricks/terraform-field-dev @databricks/eng-plat-auto-exp-reviewers
```

```suggestion
*   @databricks/field-dev-terraform @databricks/eng-plat-auto-exp-reviewers
```
to follow more standard group naming